template.rb

```diff
@@ -260,8 +260,8 @@ def gem_exists?(name)
   say
   say "  # Update config/database.yml with your database credentials"
   say
-  say "  rails g noticed:model"
   say "  rails db:create db:migrate"
+  say "  rails g noticed:model"
```

Comment:
I think you would still need to migrate after generating the model, right?

Reply:
You are right, it's better if first we suggest creating only the database, then generate noticed migrations and last make the migration to start the APP.